community/brillo/depends

```diff
@@ -0,0 +1 @@
+go
```

Comment:
Is Go still a dependency?

Reply:
Only as a `make` dependancy. With regards to `go-md2man`  lines 6, lines 70-76 eliminate the need it.

Reply:
I have pushed the new commits and updated the manifest to reflect the new changes made.